meta-mender-toradex-nxp/recipes-bsp/u-boot/files/colibri-imx7-emmc/0001-colibri-imx7-emmc-mender-integration-patch.patch

```diff
@@ -0,0 +1,71 @@
+From a0976db34d2a9ed8f65dbc5041e6f21c3b22cfa6 Mon Sep 17 00:00:00 2001
+From: Mirza Krak <mirza.krak@northern.tech>
+Date: Thu, 11 Oct 2018 16:43:11 +0200
+Subject: [PATCH 1/1] colibri-imx7-emmc: mender integration patch
+
+Signed-off-by: Mirza Krak <mirza.krak@northern.tech>
+---
+ include/configs/colibri_imx7_emmc.h | 30 +++++++++++------------------
+ 1 file changed, 11 insertions(+), 19 deletions(-)
+
+diff --git a/include/configs/colibri_imx7_emmc.h b/include/configs/colibri_imx7_emmc.h
+index 9c49f885d8..0ce63a107c 100644
+--- a/include/configs/colibri_imx7_emmc.h
++++ b/include/configs/colibri_imx7_emmc.h
+@@ -69,13 +69,12 @@
+ #define CONFIG_SERVERIP			192.168.10.1
+ 
+ #define EMMC_BOOTCMD \
+-	"emmcargs=ip=off root=/dev/mmcblk0p2 ro rootfstype=ext4 rootwait\0" \
+ 	"emmcboot=run setup; " \
+-		"setenv bootargs ${defargs} ${emmcargs} ${setupargs} " \
+-		"${vidargs}; echo Booting from internal eMMC chip...; " \
+-		"run m4boot && " \
+-		"load mmc 0:1 ${fdt_addr_r} ${soc}-colibri-emmc-${fdt_board}.dtb && " \
+-		"load mmc 0:1 ${kernel_addr_r} ${boot_file} && " \
++		"setenv bootargs '${defargs} ip=off root=${mender_kernel_root} ro rootfstype=ext4 rootwait ${setupargs} " \
++		"${vidargs}'; echo Booting from internal eMMC chip...; " \
```

Comment:
The final `${vidargs}` here is confusing, since it is inside the quotes of the previous `setenv`. Could you move it to the previous line?